source/braille.py

```diff
@@ -920,9 +927,9 @@ def update(self):
 		self._rawToContentPos = []
 		self._currentContentPos = 0
 		self.selectionStart = self.selectionEnd = None
+		self._skipFormatFieldsWithText = False
 		self._skipFieldsNotAtStartOfNode = False
 		self._endsWithField = False
-

```

Comment:
nit: Is there any reason to remove this blank line? Otherwise, let's reinstate it. :)